lib/requests/diaryFoods.js

```diff
@@ -5,6 +5,31 @@ const getDiaryFoods = () => {
     .catch(error => console.error({ error }))
 }
 
+const getMeals = () => {
+  fetch('https://ivmh-qs-api.herokuapp.com/api/v1/meals')
+    .then(response => handleResponse(response))
+    .then(meals => {
+      meals.forEach(meal => populateMealTable(meal))
+    })
+    .catch(error => console.error({ error })
+)}
+
+const populateMealTable = (meal) => {
+  meal.foods.forEach(food => addFoodToMeal(meal, food))
+}
+
+const renderFoodToMealTable = (meal, food) => {
+  $(`#${meal.name.toLowerCase()}-table-info`).append(
+    `<article class="food-item-${food.id}" id="food-item-row" data="food-${food.id}">
+      <p class="food-item-name">${food.name}</p>
+      <p class="food-item-calories">${food.calories}</p>
+      <div class="button-container">
+        <button id="food-item-${food.id}" class="food-item-delete-btn" aria-label="Delete">-</button>
```

Comment:
Can we be more specific with the naming of this delete button? Something like `food-meal-${food.id}` and then `food-meal-delete-btn`

Reply:
Specificity for this delete button is doable but if the purpose is to avoid overlapping functionality between foods and index, there shouldn't be a concern because the event listener we have for the #food-table-info is limited to only on the foods view.

Reply:
Great call. Since the eventListener is added to the overall table, we'll put a different eventListener on each specific table to send the delete request to the meals endpoint.